tests/test_python.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+
```

Comment:
The name `test_python.py` IMHO does not describe, what is going on in this file.